lib/tasks/dev.rake

```diff
@@ -1,5 +1,26 @@
 namespace :ofn do
   namespace :dev do
+    desc 'Setup dev environment'
+    task setup: :environment do
+      p '== Entering setup'
+      # NOTE: Could be helpful to drop and create again the db here
+      # Rake::Task['db:drop'].invoke
+      # Rake::Task['db:create'].invoke
+
+      unless Spree::User.table_exists? && Spree::User.count
+        p '== Load data schema'
+        Rake::Task['db:schema:load'].invoke
+
+        # TODO: Integrate the tasks loading data
+        # Issue to reach stdin while creating Admin account
+        # Rake::Task['db:seed'].invoke
+        # Rake::Task['ofn:sample_data'].invoke
+      end
+
+      p '== Migrate'
+      Rake::Task['db:migrate'].invoke
+    end
```

Comment:
I don't see the value of abstracting this. Can't we simply run `bundle exec db:setup` in `docker-compose.yml`? It does the `db:create` and `db:seed` for us. If we prefer, we could simply run `bundle exec db:create db:schema:load` instead.

Besides, there is no way around learning what each of these Rails tasks do is someone wants to develop, IMO. Hiding them does no good to new contributors.

Reply:
The value is that we may not want to rebuild the dataset each time we are launching the container, especially that there are few tests that can take time to setup at data level, and you may need to restart the container for some reasons (Data migration, dependencies loading, ...). Also, it is not straight forward to load the data today and you unfortunately need to load them manually. 

This task will provide some flexibility to the user who wants to perform some tasks launching the container, that's why I let some commented lines here too. It is not perfect, but it is a first step, I am always putting myself in the shoes of someone launching the app for the first time, and how to make it smoothly.

This was just to clarify my point of this weird task here, but if it is a real concern for you, I will adopt the strategy you propose, no problem.

Reply:
let's see what others think. I would prefer teaching new devs to fish rather than giving ourselves the fish.

Reply:
Yeah, I'd remove this. The rake tasks already exist, right? This logic can just go in the docker files. See my comment above for running `db:seed` non-interactively.